### PR TITLE
[13.0][ADD] stock_move_line_reference_link

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Stock Move Line Reference Link
=========================

This module adds link to references in stock move lines.

### CODE REVIEW

File: stock_move_line_reference_link/views/move_line_views.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 ForgeFlow S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="view_move_line_form" model="ir.ui.view">
+        <field name="name">stock.move.line.form (stock_move_line_reference_link)</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_move_line_form" />
+        <field name="arch" type="xml">
+            <field name="reference" position="replace">
```
Comment (JordiBForgeFlow) [inline stock_move_line_reference_link/views/move_line_views.xml]: instead of replace use position="attributes" to make the field invisible

Comment (JordiBForgeFlow) [reply stock_move_line_reference_link/views/move_line_views.xml]: It's not really necessary to hide it

File: stock_move_line_reference_link/views/move_line_views.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                <field name="reference" invisible="1" />
+                <field name="linked_reference" string="Reference Link" />
```
Comment (JordiBForgeFlow) [inline stock_move_line_reference_link/views/move_line_views.xml:11]: add this as 
 <field name="reference" position="after">

**Review by AdriaGForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

**Review by HaraldPanten — APPROVED** [review]
LGTM 👍

**Comment by MiquelRForgeFlow** [comment]
Moved to https://github.com/OCA/stock-logistics-workflow/pull/669.

**Pull request closed without merging**